add-chain/chain_config.go

```diff
@@ -81,6 +82,16 @@ func constructChainConfig(
 		},
 	}
 
+	if superchainLevel == superchain.Standard || standardChainCandidate {
+		var faultProofsTag string
```

Comment:
Suggest renaming this to `contractsVersionTag`. If `!isFaultProofs`, this variable name is confusing:
```
contractsVersionTag := "op-contracts/v1.3.0"
if isFaultProofs {
  contractsVersionTag = "op-contracts/v1.4.0"
}
chainConfig.ContractsVersionTag = &contractsVersionTag
```

Reply:
[ea26c10](https://github.com/ethereum-optimism/superchain-registry/pull/297/commits/ea26c10fdc4762256e31c8d8317c65915a97737d)